src/domain/occurrence/utils.ts

```diff
@@ -144,20 +146,31 @@ export const getEditEventPayload = ({
       enrolmentEndDays: Number(enrolmentEndDays) || 0,
       neededOccurrences: Number(neededOccurrences) || 0,
       autoAcceptance,
+      translations:
+        autoAcceptance && autoAcceptanceMessage
+          ? [
+              {
+                languageCode: Language.Fi,
```

Comment:
Seems odd that Finnish would be hardcoded here. Checking that it's intended.

Reply:
it's intended until the UI has a multi language support. It's not designed yet and the ticket says that it's not needed yet. However, the API already supports it, because it would be harder to make the migrations afterwards. The API also uses FI as a fallback language.